docs/markdown/Gnome-module.md

```diff
@@ -92,6 +92,49 @@ Note that if you `#include` the generated header in any of the sources for a bui
 
 Returns an array of two elements which are: `[c_source, header_file]`
 
+### gnome.mkenums_simple()
+
+Generates enum `.c` and `.h` files for GObject using the `glib-mkenums` tool
+with the standard template used by most GObject-based C libraries. The first
+argument is the base name of the output files.
+
```

Comment:
In `gnome.mkenums` you should probably reference this function and explain why you might want to use this.